### PR TITLE
chill the logs out!

### PR DESCRIPTION
**Pull request by mixmix**
This PR was made to try and fix the logging noise made by Uint8Arrays, but there is still problem upstream:

https://github.com/entropyxyz/cli/issues/160

### CODE REVIEW

File: src/common/logger.ts [hunk 1]
```diff
@@ -36,7 +39,7 @@ export class EntropyLogger {
       winston.format.splat(),
       // Uses safe-stable-stringify to finalize full object message as string
       // (prevents circular references from crashing)
-      winston.format.json(),
+      winston.format.json({ replacer: replacer }),
```
Comment (rh0delta) [inline src/common/logger.ts:42]:
```suggestion
      winston.format.json({ replacer }),
```

File: src/common/utils.ts [hunk 2]
```diff
@@ -13,16 +13,21 @@ export function stripHexPrefix (str: string): string {
 
 export function debug (...args: any[]) {
   _debug(...args.map(arg => {
-    return typeof arg === 'object'
-      ? JSON.stringify(arg, replacer, 2)
-      : arg
+    return stringify(arg, 2)
   }))
 }
```
Comment (rh0delta) [inline src/common/utils.ts:18]: i think we are going to move away from using this utility for logs

Comment (mixmix) [reply src/common/utils.ts:18]: yeah that can be part of your PR I think
- replace `debug` and `print` functions
- uninstall `debug` module

Comment (rh0delta) [reply src/common/utils.ts:18]: Sounds good to me!

**Comment by rh0delta** [comment]
@mixmix you think we need a migration to update how we are setting the config through config.set? I believe if we update stringification there we should be able to get away from needing a migration because of how we are subscribing to updates. The data gets refreshed everytime, so it should get fixed once the method is updated with your new replacer method

**Comment by mixmix** [comment]
I'm worried it is desrializing wrong. Frankie said it's likely only working cos we use the seed not the secretKey.
i've got it half written today in a branch (pr later)

**Review by rh0delta — APPROVED** [review]

**Comment by mixmix** [comment]
well that PR collapsed down nice!
ready for another review @rh0delta / smoke test

**Comment by mixmix** [comment]
@rh0delta GO TO BED
also thanks for the reviews. AND if you review and approve feel free to merge!  (unless there is some "wait till X is in" note)

:heart: :zzz:

**Pull request merged**